contrib/kubernetes/rc/r1.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: etcd-1
+spec:
+  replicas: 1
+  selector:
+    app: etcd-1
+  template:
+    metadata:
+      labels:
+        app: etcd-1
+        service: etcd
+    spec:
+      containers:
+        - name: etcd
+          image: gcr.io/google_containers/etcd:2.0.12
```

Comment:
turns out that the quay.io image is really really bare bones.  no shell which makes it hard to run.  going back to gcr image.